src/screencapturekit.cs

```diff
@@ -335,4 +453,105 @@ interface SCStreamOutput {
 		[Export ("stream:didOutputSampleBuffer:ofType:")]
 		void DidOutputSampleBuffer (SCStream stream, CMSampleBuffer sampleBuffer, SCStreamOutputType type);
 	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	interface SCContentSharingPickerConfiguration {
+		[Export ("allowedPickerModes", ArgumentSemantic.Assign)]
+		SCContentSharingPickerMode AllowedPickerModes { get; set; }
+
+		[Export ("excludedWindowIDs", ArgumentSemantic.Strong)]
+		NSNumber [] ExcludedWindowIDs { get; set; }
+
+		[Export ("excludedBundleIDs", ArgumentSemantic.Strong)]
+		string [] ExcludedBundleIDs { get; set; }
+
+		[Export ("allowsChangingSelectedContent")]
+		bool AllowsChangingSelectedContent { get; set; }
+	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SCContentSharingPicker {
+		[Static]
+		[Export ("sharedPicker")]
+		SCContentSharingPicker SharedPicker { get; }
+
+		[Export ("defaultConfiguration", ArgumentSemantic.Copy)]
+		SCContentSharingPickerConfiguration DefaultConfiguration { get; set; }
+
+		[NullAllowed]
+		[Export ("maximumStreamCount", ArgumentSemantic.Strong)]
+		NSNumber MaximumStreamCount { get; set; }
```

Comment:
Probably we can use [BindAs](https://learn.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#bindasattribute) here according to [docs](https://developer.apple.com/documentation/screencapturekit/sccontentsharingpicker/4274057-maximumstreamcount)

```suggestion
		[NullAllowed]
		[BindAs (typeof (int))]
		[Export ("maximumStreamCount", ArgumentSemantic.Strong)]
		NSNumber MaximumStreamCount { get; set; }
```